### DISCUSSION TITLE
CMD to create table  bulk import [Apr 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1839] [Discord Threads]
> 🙋‍♂️ **vishal.gandhi**

**Comment by o1lab** [comment]
> 🙋‍♂️ **vishal.gandhi**  
#### hello wingkwong  
#  
> 👨‍🚀 **wingkwong**  
#### if u go to swagger ui, then execute a api call, you will see the curl command  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### trying  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### how do i go to swagger UI  
#  
> 👨‍🚀 **wingkwong**  
#### assuming you r using rest api project, check this out  
#  
> 👨‍🚀 **wingkwong**  
#### https://docs.nocodb.com/developer-resources/accessing-apis#rest-apis  
#  
> 👨‍🚀 **wingkwong**  
#### after executing an api call, u should see sth like  
![image0](https://cdn.discordapp.com/attachments/964016215066566746/964019724935647272/unknown.png)  
#  
> 👨‍🚀 **wingkwong**  
#### with similar idea, you may check out https://docs.nocodb.com/developer-resources/rest-apis#examples  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### Able to access APIs now. I will try this out. Thank you.  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### however is there API for create table  
#  
> 👨‍🚀 **wingkwong**  
#### api for create table will be avilable in the next release  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### awesome thank you so much  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### i am trying to import CSV and its failing with 400  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### .returning() is not supported by mysql and will not have any effect.   
#### POST /nc/work_gkos/api/v1/planview/bulk 400 17168 - 22.880 ms   
#### .returning() is not supported by mysql and will not have any effect.   
#### POST /nc/work_gkos/api/v1/planview/bulk 400 17168 - 28.775 ms   
#### .returning() is not supported by mysql and will not have any effect.   
#### POST /nc/work_gkos/api/v1/planview/bulk 400 17168 - 13.598 ms  
#  
> 🙋‍♂️ **vishal.gandhi**  
#### this is as per docker container logs  
#  
> 👨‍🚀 **wingkwong**  
#### pls create an issue on github with some details (project info, steps to reproduce etc)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/964016215066566746)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
